x-pack/plugins/lens/public/datatable_visualization/components/toolbar.tsx

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { EuiFlexGroup, EuiFormRow, EuiSwitch } from '@elastic/eui';
+import { ToolbarPopover } from '../../shared_components';
```

Comment:
nit: we try to add `type` when importing types. Please use this pattern

Reply:
done